estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class EstatePropertyTag(models.Model):
+    _name = 'estate.property.tag'
+    _description = 'Estate Property Tag'
+    _order = "name"
```

Comment:
little nitpick, although not everybody respects that at odoo, we have somewhat of a convention for using single vs double quotes. Everything that is user-facing (e.i. everything an end-user will see) is usually double-quoted. Everything else is single quoted (so mostly technical strings). 
```suggestion
    _description = "Estate Property Tag"
    _order = 'name'
```